packages/capabilities/test/capabilities/access.test.js

```diff
@@ -371,3 +371,256 @@ describe('access capabilities', function () {
     })
   })
 })
+
+describe('access/delegate', () => {
+  it('can create valid delegations and authorize them', async () => {
+    const issuer = alice
+    const audience = service.withDID('did:web:web3.storage')
+    const bobCanStoreAllWithAlice = await delegate({
+      issuer: alice,
+      audience: bob,
+      capabilities: [{ can: 'store/*', with: alice.did() }],
+    })
+    const validInvocations = [
+      // uncommon to have empty delegation set, but it is valid afaict
+      Access.delegate
+        .invoke({
+          issuer,
+          audience,
+          with: issuer.did(),
+          nb: {
+            delegations: {},
+          },
+        })
+        .delegate(),
+      // example from spec, but once each with several different, but all valid, property names to use in the `nb.delegations` dict
+      // https://github.com/web3-storage/specs/blob/7e662a2d9ada4e3fc22a7a68f84871bff0a5380c/w3-access.md?plain=1#L58
+      .../** @type {const} */ ([
+        // correct cid
+        [bobCanStoreAllWithAlice.cid.toString(), bobCanStoreAllWithAlice.cid],
+        // not a cid at all
+        ['thisIsNotACid', bobCanStoreAllWithAlice.cid],
```

Comment:
🤔 I'm starting to wonder if the whole dict is a good idea here, but then if we chose list it also may not be ordered.